src/main/java/uk/gov/companieshouse/authcodenotification/controller/AuthCodeNotificationController.java

```diff
@@ -38,15 +48,29 @@ public ResponseEntity<Object> sendEmail(@RequestHeader(value = ERIC_REQUEST_ID_K
                                             @PathVariable String companyNumber) {
 
         companyNumber = dataSanitiser.makeStringSafe(companyNumber);
```

Comment:
What happens when a null or empty string is passed into the 'sanitiser' for company name and/or auth code? Looks like we don't have any specific tests around this. Ideally we'd add them to the controller unit test class, to be sure of behaviour.

Reply:
Added 3 tests null, empty and blank to the DataSanitiser unit tests and find that the owasp encoder just return a string of whatever is input so we get:

"null"
""
"  "

for:

null
""
"  "